### PR TITLE
Mergeback 1.1.0

### PR DESCRIPTION
**Pull request by rtisma**

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.0.0
+1.1.0
```
Comment (lepsalex) [inline VERSION:1]: 1.1.1-SNAPSHOT?

**Review by lepsalex — CHANGES_REQUESTED** [review]

File: VERSION (hunk 1 above)
Comment (rtisma) [reply VERSION:1]: SNAPSHOT is how maven does things. If you look at the Jenkinsfile, merges on the develop branch have this syntax
`<version>-<commitid>` so its effectively the same thing.

Comment (lepsalex) [reply VERSION:1]: still needs to be 1.1.1 no, develop should be ahead of master

Comment (rtisma) [reply VERSION:1]: ok ill change to snapshot since its more clear

**Comment by rtisma** [comment]
closing this to make a proper mergeback pr

**Pull request closed without merging**